src/main/resources/logback.xml

```diff
@@ -42,6 +42,12 @@
         <appender-ref ref="SYSLOG"/>
     </logger>
 
+    <logger name="org.broadinstitute.dsde.rawls.dataaccess.HttpGooglePubSubDAO" level="debug" additivity="false">
+        <appender-ref ref="file"/>
+        <appender-ref ref="console"/>
+        <appender-ref ref="SYSLOG"/>
+    </logger>
+
```

Comment:
which on second thought is going to be a crap load of things logged if we're pulling messages off one at a time